scripts/samples.sh

```diff
@@ -95,52 +95,18 @@ else
     echo "No NAMESPACE specified - may share assets etc with someone else on same URL"
 fi
 
-DOOR_ENTRY="${TEST_NO_DOOR_ENTRY} python3 -m archivist_samples.door_entry ${ARGS} ${NAMESPACE}"
-${DOOR_ENTRY} --create
-${DOOR_ENTRY} --list all
-${DOOR_ENTRY} --list doors
-${DOOR_ENTRY} --list cards
-${DOOR_ENTRY} --list 'Courts of Justice front door'
-${DOOR_ENTRY} --list 'access_card_1'
-
-OPEN="${DOOR_ENTRY} --open"
-${OPEN} "Courts of Justice front door,access_card_1"
-${OPEN} "Courts of Justice front door,access_card_3"
-${OPEN} "Courts of Justice front door,access_card_4"
-${OPEN} "Courts of Justice front door,access_card_0"
-${OPEN} "Courts of Justice front door,access_card_2"
-${OPEN} "Bastille front door,access_card_2"
-${OPEN} "City Hall front door,access_card_2"
-${OPEN} "Gare du Nord apartments side door,access_card_2"
+${TEST_NO_DOOR_ENTRY} python3 -m archivist_samples.door_entry ${ARGS} ${NAMESPACE}
```

Comment:
What's this doing? It looks like it inserts a colon before this sometimes?

Reply:
yes - it is a shorthand way of not executing something in bash